Automattic/woocommerce-payments
# Add a new note as a placeholder for UPE CTA

Fixes https://github.com/Automattic/woocommerce-payments/issues/2168

#### Changes proposed in this Pull Request
This PR adds an admin note when `_wcpay_feature_upe_settings_preview` is enabled but `_wcpay_feature_upe` is disabled. 

When clicked "Enable on your store", it should enable the UPE feature flag by setting `_wcpay_feature_upe` to 1.

#### Testing instructions
1. Enable UPE settings preview by setting `_wcpay_feature_upe_settings_preview` to 1 or make [this function](https://github.com/Automattic/woocommerce-payments/blob/06ad35edec56d92e87d6d2f9a22bf39b81ffc403/includes/class-wc-payments-features.php#L67) return true.
2. Make sure `_wcpay_feature_upe` doesn't exist in your DB. You can check with `SELECT * FROM `wp_options` where option_name = '_wcpay_feature_upe'`
3. Go to Home `wp-admin/admin.php?page=wc-admin` or Orders `wp-admin/edit.php?post_type=shop_order`, click on "Inbox" and you should see the admin note:
![image](https://user-images.githubusercontent.com/572862/123132221-ba48e400-d40b-11eb-9b74-75b32a4d6ff4.png)

4. Click "Enable on your store", check database using query in step 2 and to confirm the feature flag is added. It should redirects you to `wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments`. 
5. If you would like, you can repeat the above steps for the matrix below:

| _wcpay_feature_upe_settings_preview | _wcpay_feature_upe | Show Admin Note?|
|--|--|--|
|true|true| no|
|true|false| yes |
|false|true| no|
|false|false| no|

### TODO:
- Update learn more link, pending. We can make a follow-up issue

-------------------
